### ISSUE TITLE
class-name-format not working with variable interpolation

### ISSUE THREAD

**Issue by ncoden** [sasstools/sass-lint#952]
I am using `gulp-sass-lint v1.3.2` with `sass-lint v1.10.2`.

For the following code:
```scss
    // Static (unresponsive) row gutters
    @each $breakpoint, $value in $grid-column-gutter {
      &.#{$gutter}-#{$breakpoint} {
        > .#{$column} {
          @include grid-col-gutter($value);
        }
      }
    }
```
(from [foundation-sites / grid/_classes.scss](https://github.com/zurb/foundation-sites/blob/546a5b9f38487450df0dcf4957ba8c73bf11194e/scss/grid/_classes.scss#L59))

The following warning is throw on `&.#{$gutter}-#{$breakpoint}`:
```
   59:19  warning  Class '.-' should match regular expression /([a-z0-9]+-?)+/  class-name-format
```

The sass-lint config is :
```yml
...
  class-name-format:
    - 1
    - convention: '([a-z0-9]+-?)+'
...
```
(from [.sass-lint.yml](https://gist.github.com/ncoden/ec50bad6cd2f08df0780e9f356442794))

Sass-lint seems to ignore variable interpolation, and so consider the variable name as `.-`. I expect `class-name-format` to consider the class name as `\.(.*)\-(.*)` for the regex check.

**Comment by DanPurdy** [comment]
I'll double check this, shouldn't be throwing any issues here.

**Comment by coreyjv** [comment]
Is there an update on this bug?

**Comment by ncoden** [comment]
Poke @DanPurdy

**Comment by DanPurdy** [comment]
Sorry for this insane delay, life getting in the way of things... I will check this in #1131 as we should be ignoring interpolated classnames in the next release.

**Comment by DanPurdy** [comment]
Yup, just run it in the tests this doesn't throw the error anymore. WIll mark it as fixed with that PR but give it a final test before we release it. Sorry again for the delay.

**Comment by DanPurdy** [comment]
Released in 1.12.0
